airflow/contrib/hooks/gcp_api_base_hook.py

```diff
@@ -45,6 +50,42 @@
 _G_APP_CRED_ENV_VAR = "GOOGLE_APPLICATION_CREDENTIALS"
 
 
+# Constants used by the mechanism of repeating requests in reaction to exceeding the temporary quota.
+INVALID_KEYS = [
+    'DefaultRequestsPerMinutePerProject',
+    'DefaultRequestsPerMinutePerUser',
+    'RequestsPerMinutePerProject',
+    "Resource has been exhausted (e.g. check quota).",
+]
+INVALID_REASONS = [
+    'userRateLimitExceeded',
+]
+
+
+# pylint: disable=invalid-name
```

Comment:
Doesn't it cause that  invalid names are disabled for all lines below?

Reply:
Yes. You're right. This should be in line with the warning.